### DISCUSSION TITLE
how to deploy graphclient server to lambda/cloudflare workers

### DISCUSSION THREAD

**Discussion by ethdev279** [graphprotocol/graph-client#236] [General]
I'm trying to deploy the graph client server instance to serverless environments/elastic beanstalk. can anyone pls let me know a way to do it.  we need to build it and serve right? I know graph client uses mesh under the hood. so I was looking at this: https://www.the-guild.dev/graphql/mesh/docs/getting-started/deploy-mesh-gateway

**Comment by dotansimha** [comment]
Hi @ethdev279 !
You are right, Graph Client is using Mesh under the hood. So you can run `graph-client build` and get the (Mesh) artifact created for you under `.graphclient` dir.

Then you can just use the Mesh artifact as-is. For a CF Worker, you can do: 

```ts
import { createBuiltMeshHTTPHandler } from './.graphclient'
 
// Pass Mesh's HTTP handler as an event listener
self.addEventListener('fetch', createBuiltMeshHTTPHandler())
```

(same for any other Mesh example in https://www.the-guild.dev/graphql/mesh/docs/getting-started/deploy-mesh-gateway ) 

> Please make sure you are using the latest `graph-client` to have the `createBuiltMeshHTTPHandler` function in the generated artifact :)

**Reply by ethdev279** [reply]
Thanks for your response. Tried it. it seems like there's an issue now

```
D:\ethdev279\uniswap-graph-client\.graphclient\index.ts:1084
    getBuiltMesh,
    ^
ReferenceError: getBuiltMesh is not defined
```
Missing declaration here or am I missing anything?
```
export function createBuiltMeshHTTPHandler() {
  return createMeshHTTPHandler({
    baseDir,
    getBuiltMesh,
    rawServeConfig: undefined,
  })
}
```

**Reply by ardatan** [reply]
You're right. That factory function was broken, and it has been fixed in the latest release. Could you confirm after updating the packages to the latest?

**Reply by ethdev279** [reply]
Thanks, it seems working. but with an error/warning:

```
💥 GraphClient GraphQLError: GraphQL only supports GET and POST requests
```

**Comment by saihaj** [comment]
Here is an example https://github.com/the-guild-org/maverick-subgraph
